### PR TITLE
check before filling issues

### PR DESCRIPTION
**Pull request by Alokpro1**

### CODE REVIEW

**Comment by bobbrow** [comment]
I believe this page is about contributing to the code.  This change might be better placed in the issue templates.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -3,7 +3,8 @@
 ## Contribution Steps
 
 * [Build and debug the extension](Documentation/Building%20the%20Extension.md).
-* File an [issue](https://github.com/Microsoft/vscode-cpptools/issues) and a [pull request](https://github.com/Microsoft/vscode-cpptools/pulls) with the change and we will review it.
+* Before filling a new issue **check if the issue already exists in it.**
```
Comment (sean-mcmanus) [inline CONTRIBUTING.md]: Should be spelled "filing".

Comment (sean-mcmanus) [inline CONTRIBUTING.md]: I think "in it" should be removed. I don't understand what "it" is referring to.

**Comment by sean-mcmanus** [comment]
Bob is referring to https://github.com/Microsoft/vscode-cpptools/tree/master/.github/ISSUE_TEMPLATE , which gets shown to users when the file a new issue.

File: CONTRIBUTING.md (hunk 1 above)
Comment (Alokpro1) [reply CONTRIBUTING.md]: okay. Sorry for the mistake

**Comment by sean-mcmanus** [comment]
@Alokpro1 The language service issue template already has ``, so I don't think we need to add this to the contributing.md.

**Pull request closed without merging**